.tools/psalm/baseline.xml

```diff
@@ -417,6 +420,9 @@
     </MixedOperand>
   </file>
   <file src="redaxo/src/core/fragments/core/fe_ooops.php">
+    <InvalidScope>
+      <code><![CDATA[$this]]></code>
```

Comment:
Hier stimmt was nicht. Kann es sein, dass die `@psalm-scope-this`-Annotations nicht angepasst wurden in den Fragmenten?

Reply:
Wurden Sie, aber das use muss vor den Kommentare stehen.